src/main/java/seedu/address/model/session/SessionList.java

```diff
@@ -122,6 +123,39 @@ public void remove(Session toRemove) {
         }
     }
 
+    /**
+     * Removes the {@code student} from all sessions in the sessions list.
```

Comment:
Not sure if this removes student from ALL session, or just 1 session

Reply:
It removes the student from all sessions in the sessions list. The idea is to map remove functions on each `Session` object so there is no more instance of the student in anywhere in the sessions of the list.